### ISSUE TITLE
a sudden shutdown of program during the paratroopers action

### ISSUE THREAD

**Issue by titaniumman** [triplea-game/triplea#2776]
[bugreport2.zip](https://github.com/triplea-game/triplea/files/1595032/bugreport2.zip)

### Can you describe how to trigger the error? (eg: what sequence of actions will recreate it?)
paratroopers, long range aircraft activated
in German combat move phase
Low Luck in AAgun

from morocco to eastern united states
I did group selecting with 17 bombers and 16infantries together at once, and sent them together and then when I conduct the combat  in eastern united states,
as America has 2AAgun in the eastern united states, so as Lowluck, 1aircraft must get crash and falldown immediately before the battle
so i sent 17bombers and 16infantries. because when paratroopers conducting, when enemy's AAgun hit the infantries also die immediately.
when choose AAgun casualty  i choose the empty bomber which didn't bring with infantry
and click for casualty decision. at the very moment suddenly the console window pop up and game got shutdown as error

### Do you have the exact error text? Please copy/paste if so
```
triplea.engine.version.bin:1.9
Loading map: world_war_ii_v5_1942, from: C:\Users\a\triplea\downloadedMaps\world_war_ii_v5_1942-master.zip
Loading resources from the following paths: [C:\Users\a\triplea\downloadedMaps\world_war_ii_v5_1942-master.zip, C:\Program Files\TripleA\assets]
Loading map: world_war_ii_v5_1942, from: C:\Users\a\triplea\downloadedMaps\world_war_ii_v5_1942-master.zip
Loading resources from the following paths: [C:\Users\a\triplea\downloadedMaps\world_war_ii_v5_1942-master.zip, C:\Program Files\TripleA\assets]
java.lang.IllegalStateException: Not being carried, unit:infantry owned by Germans transport:bomber owned by Germans
	at games.strategy.triplea.delegate.TransportTracker.unloadAirTransportChange(TransportTracker.java:162)
	at games.strategy.triplea.delegate.MustFightBattle.landParatroops(MustFightBattle.java:2298)
	at games.strategy.triplea.delegate.MustFightBattle.access$700(MustFightBattle.java:52)
	at games.strategy.triplea.delegate.MustFightBattle$9.execute(MustFightBattle.java:724)
	at games.strategy.triplea.delegate.ExecutionStack.execute(ExecutionStack.java:41)
	at games.strategy.triplea.delegate.MustFightBattle.fight(MustFightBattle.java:384)
	at games.strategy.triplea.delegate.BattleDelegate.fightBattle(BattleDelegate.java:219)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.delegate.DelegateExecutionManager$1.invoke(DelegateExecutionManager.java:156)
	at com.sun.proxy.$Proxy4.fightBattle(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeSingle(EndPoint.java:164)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeMultiple(EndPoint.java:146)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeLocal(EndPoint.java:132)
	at games.strategy.engine.message.unifiedmessenger.UnifiedMessenger.invokeAndWait(UnifiedMessenger.java:103)
	at games.strategy.engine.message.UnifiedInvocationHandler.invoke(UnifiedInvocationHandler.java:51)
	at com.sun.proxy.$Proxy4.fightBattle(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.gamePlayer.DefaultPlayerBridge$GameOverInvocationHandler.invoke(DefaultPlayerBridge.java:154)
	at com.sun.proxy.$Proxy4.fightBattle(Unknown Source)
	at games.strategy.triplea.TripleAPlayer.battle(TripleAPlayer.java:480)
	at games.strategy.triplea.TripleAPlayer.start(TripleAPlayer.java:129)
	at games.strategy.engine.framework.ServerGame.waitForPlayerToFinishStep(ServerGame.java:524)
	at games.strategy.engine.framework.ServerGame.runStep(ServerGame.java:410)
	at games.strategy.engine.framework.ServerGame.startGame(ServerGame.java:266)
	at games.strategy.engine.framework.startup.launcher.LocalLauncher.launchInNewThread(LocalLauncher.java:61)
	at games.strategy.engine.framework.startup.launcher.AbstractLauncher.lambda$launch$164(AbstractLauncher.java:51)
	at java.lang.Thread.run(Unknown Source)
	at games.strategy.engine.message.UnifiedInvocationHandler.invoke(UnifiedInvocationHandler.java:63)
	at com.sun.proxy.$Proxy4.fightBattle(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.gamePlayer.DefaultPlayerBridge$GameOverInvocationHandler.invoke(DefaultPlayerBridge.java:154)
	at com.sun.proxy.$Proxy4.fightBattle(Unknown Source)
	at games.strategy.triplea.TripleAPlayer.battle(TripleAPlayer.java:480)
	at games.strategy.triplea.TripleAPlayer.start(TripleAPlayer.java:129)
	at games.strategy.engine.framework.ServerGame.waitForPlayerToFinishStep(ServerGame.java:524)
	at games.strategy.engine.framework.ServerGame.runStep(ServerGame.java:410)
	at games.strategy.engine.framework.ServerGame.startGame(ServerGame.java:266)
	at games.strategy.engine.framework.startup.launcher.LocalLauncher.launchInNewThread(LocalLauncher.java:61)
	at games.strategy.engine.framework.startup.launcher.AbstractLauncher.lambda$launch$164(AbstractLauncher.java:51)
	at java.lang.Thread.run(Unknown Source)
```

### Instead of this error, what should have happened?

shouldn't be shutdown and battle should have been conducted normally

**Comment by ron-murhammer** [comment]
@titaniumman I was able to reproduce the error. There appears to be some issue with moving unloaded air transport along with loaded air transports in the same move and then having it die in combat. If you undo the air transport move and move the 26 bombers with 26 inf first then move the remaining empty bomber and play the battle then you won't get the error.

**Comment by ron-murhammer** [comment]
@titaniumman This should now be resolved with the latest pre-release if you follow these steps. Load save from above bug report. Undo move 13 (17 bomber & 16 inf) and then perform the move again. Then follow battle reproduction steps and you shouldn't see an error. I'll leave this open for a few days then close it if no one reports any issues with it.

**Comment by titaniumman** [comment]
hi sir. thank you for comment and helping. i had tried in many other ways as experiments by myself and could avoid the crash by the turning-around method as you advised. but still it was an abnormal feature of crash and i wanted to report, somehow contribute for this blessing open source game which was made for everyone for free in addition to the impressive efforts of developers regardless of their non-beneficial work. haven't reported any bug at other applications in my life since it's also a bit handy to write and attach files for an IT newbie but got some special feeling on this game :) so would you let me know whether or not this bug will be going to be considered as something to fix?
thank you again for your help and attention, and happy new year!
best wishes

**Comment by RoiEXLab** [comment]
@titaniumman This issue has already been fixed in  version 1.9.0.0.8298
You can find all pre-releases [here](https://github.com/triplea-game/triplea/releases)
